MoveCardAfterReview/ReadMe.md

```diff
@@ -14,13 +18,60 @@ Token to access GitHub API.
 
 ### `column-id`
 
-ID of the Kanban column where the card should be moved to. Typically ID of `In progress` or `Review approved` column.
+ID of the Kanban column where the card should be moved to. Typically ID of `Review in progress` column. It is a number for Project classic and a string for V2. [This page](../docs/github.md) explains how this can be obtained.
```

Comment:
This looks like copy/paste issue
```suggestion
ID of the Kanban column where the card should be moved to. Typically ID of `In progress` or `Review approved` column. It is a number for Project classic and a string for V2. [This page](../docs/github.md) explains how this can be obtained.
```

Reply:
```suggestion
ID of the Kanban column where the card should be moved to. Typically the ID of the `In progress or Review approved` column. It is a number for Project classic and a string for V2. [This page](../docs/github.md) explains how this can be obtained.
```